grammar.js

```diff
@@ -988,6 +989,11 @@ module.exports = grammar({
       prec(-1, 'return'),
     ),
 
+    yield_expression: $ => choice(
+      prec.left(seq('yield', $._expression)),
+      prec(-1, 'yield'),
```

Comment:
would be nice to comment why we need a prec(-1) here, which ambiguity does it resolve?
with example of code showing the problem.